tests/mgbench/zip_mgbench.py

```diff
@@ -0,0 +1,38 @@
+import zipfile
+from pathlib import Path
+
+import log
+
+
+def zip_mgbench():
+    log.info("Creating mgbench.zip ...")
+    parent = Path(__file__).resolve().parent
+    zip = zipfile.ZipFile("./mgbench.zip", "w")
+    zip.write(parent / "benchmark.py", "mgbench/benchmark.py")
+    zip.write(parent / "setup.py", "mgbench/setup.py")
+    zip.write(parent / "log.py", "mgbench/log.py")
+    zip.write(parent / "benchmark_context.py", "mgbench/benchmark_context.py")
+    zip.write(parent / "validation.py", "mgbench/validation.py")
+    zip.write(parent / "compare_results.py", "mgbench/compare_results.py")
+    zip.write(parent / "runners.py", "mgbench/runners.py")
+    zip.write(parent / "helpers.py", "mgbench/helpers.py")
+    zip.write(parent / "graph_bench.py", "mgbench/graph_bench.py")
+    zip.write(parent / "README.md", "mgbench/README.md")
+    zip.write(parent / "how_to_use_mgbench.md", "mgbench/how_to_use_mgbench.md")
+    zip.write(parent / "workloads/base.py", "mgbench/workloads/base.py")
+    zip.write(parent / "workloads/demo.py", "mgbench/workloads/demo.py")
+    zip.write(parent / "workloads/ldbc_bi.py", "mgbench/workloads/ldbc_bi.py")
+    zip.write(parent / "workloads/ldbc_interactive.py", "mgbench/workloads/ldbc_interactive.py")
+    zip.write(parent / "workloads/pokec.py", "mgbench/workloads/pokec.py")
```

Comment:
Check all python files, ideally we would have 0 third-party deps because then user doesn't have to install any additional package during benchgraph installation, Python 3.6 as a minimum is 100% fine

Reply:
There are 0 third-party dependencies at the moment. The lowest Python version tested is 3.7 at the moment.